custom_components/alexa_media/__init__.py

```diff
@@ -266,9 +271,13 @@ async def login_success(event=None) -> None:
             "websocket": None,
             "auth_info": None,
             "second_account_index": 0,
+            "alexa_entities": (),
```

Comment:
We should probably define what would be appropriate to include in this. Are we saying anything returned as part of this API call? Also, are you storing a tuple? Shouldn't it be a dict? Note my later comment to get typing added to the declarations.

Reply:
Yes, it should have been a dict. My intention was that it should include the alexa smart home entities that can be successfully turned into HA entities. So, all the parsing/filtering/categorization of the entities happens before it is stored and it doesn't include the media players as they are very different.

Reply:
If this is the raw json data from the API, we currently store that in the `devices` dict (probably should have been named as json but that was a historical decision based off of the API call). Would it be possible to add it there? It's ok that it's filtered down to valid items.